service/controller/resource/monitoring/verticalpodautoscaler/resource.go

```diff
@@ -165,7 +165,25 @@ func (r *Resource) getMaxCPU(ctx context.Context) (*resource.Quantity, error) {
 		return nil, microerror.Mask(nodeCpuNotFoundError)
 	}
 
-	q, err := quantityMultiply(nodeCpu, 0.9)
+	n, ok := nodeCpu.AsInt64()
+	if !ok {
+		return nil, microerror.Maskf(quantityConversionError, nodeCpu.String())
+	}
+
+	var q *resource.Quantity
+	// If allocatable cpu is less then 6
+	// maxAllowedCPU is 70%
+	if n <= 6 {
```

Comment:
As @hervenicol said, I'd like an explanation on those magical numbers. Ideally a link to some documentation maybe inside PMO